stdlib/public/core/LazyCollection.swift

```diff
@@ -20,6 +20,11 @@
 /// methods that return lazy wrappers that are themselves
 /// `LazyCollectionType`s.
 ///
+/// - Note: Many lazy collections may not offer the usual performance given by
+///   `CollectionType`, `ForwardIndexType`, or `BidirectionalIndexType`. Be
+///   aware, therefore, that general operations on `LazyCollectionType`
+///   instances may not have the documented complexity.
```

Comment:
Ok. I put it here because I figured any newly-added lazy collections have a decent chance of also having unpredictable performance, but you're right, with only one actual instance of a `LazyCollectionType` with that performance, it's probably not worth having this note here.